docs/release-announcement.md

```diff
@@ -6,33 +6,45 @@ We are pleased to announce the release of version 2.8.4 of our Industrial IoT Pl
 
 ### IMPORTANT - PLEASE READ
```

Comment:
should we update this to 2.8.5? and have a separate chapter 2.8.4?

Reply:
Good catch.